translations/en-us.yaml

```diff
@@ -1,6 +1,6 @@
 oss-components:
   upf-table:
-    pagination: "{from} to {to} out of {total} {item_name}"
+    pagination: '{from} to {to} out of {total} {item_name}'
```

Comment:
Why changing quote type?

Reply:
auto linting from something on my machine ^^
Do you want me to rollback the changes ?

Reply:
If it is work, it is ok for me